api/v1/openapi.yaml

```diff
@@ -2821,6 +2830,16 @@ definitions:
       forwarded:
         description: Number of messages forwarded
         type: integer
+  IPV6BigTCP:
```

Comment:
Nit:

```suggestion
  IPv6BigTCP:
```

Reply:
this was changed automatically by something that checks the file, I had it as IPv6BigTCP and it failed a test here